src/components/Header/Header.css

```diff
@@ -0,0 +1,55 @@
+.header {
+  background: #D31027;
+  background: -webkit-linear-gradient(to right, #FF0000, #CC0000);
+  background: linear-gradient(to right, #FF0000, #CC0000);
+  height: 255px;
+  width: 100%;
+  position: relative;
+  display: flex;
+  justify-content: space-around;
+}
+
+.nav {
+  /* border: solid black 3px; */
+  margin-top: 15px;
+  width: 200px;
+  font-family: 'PocketMonk';
+  color: white;
+}
+
+.nav-links {
+  list-style: none;
+}
+
+.how-to {
+  font-size: 30px;
+}
+
+.header-image {
+  margin-top: 20px;
+  margin-right: 10px;
+  height: 230px;
+  width: 165px;
+}
+
+
+.custom-shape-divider-bottom-1631589819 {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    overflow: hidden;
+    line-height: 0;
+    transform: rotate(180deg);
+}
+
+.custom-shape-divider-bottom-1631589819 svg {
+    position: relative;
+    display: block;
+    width: calc(100% + 1.3px);
+    height: 100px;
+}
+
+.custom-shape-divider-bottom-1631589819 .shape-fill {
+    fill: #FFFFFF;
+}
```

Comment:
If we can would be cool to to some refactoring in the css / so we don't have to do that only by the end ( ordered list of name and values)